src/assets/js/contact-form.js

```diff
@@ -78,15 +83,15 @@ export class ContactForm {
     }
 
     let { action, method } = this.form.dataset;
+    action = new URL(action);
 
     let params = {
       cache: "no-cache",
       method,
       mode: "cors",
     };
-    if (method.toLowerCase() === "get") {
-      const queryString = new URLSearchParams(formData).toString();
-      action = `${action}?${queryString}`;
+    if (method.toLowerCase().match(/get/)) {
```

Comment:
I think `includes` would be a bit more appropriate.
```suggestion
    if (method.toLowerCase().includes("get")) {
```